owners.html

```diff
@@ -1,161 +1,151 @@
+    // https://stackoverflow.com/a/800010/1775459
+    document.addEventListener("DOMContentLoaded", function () {
+      var element = document.getElementById("text-open");
+      if (element) {
+        var kaliningradHours = new Date().getUTCHours() + 2;
+        var isMallOpened = kaliningradHours >= 9 && kaliningradHours < 19;
+
+        if (isMallOpened) {
+          element.classList.remove("d-none");
         }
-      });
-    </script>
+      }
+    });
+
+    //Открытие и закрытие меню по нажатию на "гамбургер"
+    var menuBtn = document.getElementById("menuBtn");
+    var menu = document.getElementById("menu");
+    menuBtn.addEventListener('click', function (event) {
+      if (menu.classList.contains("d-none")) {
+        menu.classList.remove("d-none");
+      } else {
+        menu.classList.add("d-none");
+      }
+      event.stopPropagation();
+    });
+    //закрытие меню, если оно открыто кликом по пустому месту
+    document.addEventListener("click", function (event) {
+      if (!menu.classList.contains("d-none")) {
+        menu.classList.add("d-none");
+      }
+    });
+  </script>
+
+  <!-- Yandex.Metrika counter -->
+  <script type="text/javascript">
+    (function (m, e, t, r, i, k, a) {
+      m[i] =
+        m[i] ||
+        function () {
+          (m[i].a = m[i].a || []).push(arguments);
+        };
+      m[i].l = 1 * new Date();
+      (k = e.createElement(t)),
+      (a = e.getElementsByTagName(t)[0]),
+      (k.async = 1),
+      (k.src = r),
+      a.parentNode.insertBefore(k, a);
+    })(
+      window,
+      document,
+      "script",
+      "https://mc.yandex.ru/metrika/tag.js",
+      "ym"
+    );
 
-    <!-- Yandex.Metrika counter -->
-    <script type="text/javascript">
-      (function(m, e, t, r, i, k, a) {
-        m[i] =
-          m[i] ||
-          function() {
-            (m[i].a = m[i].a || []).push(arguments);
-          };
-        m[i].l = 1 * new Date();
-        (k = e.createElement(t)),
-          (a = e.getElementsByTagName(t)[0]),
-          (k.async = 1),
-          (k.src = r),
-          a.parentNode.insertBefore(k, a);
-      })(
-        window,
-        document,
-        "script",
-        "https://mc.yandex.ru/metrika/tag.js",
-        "ym"
-      );
+    ym(53046907, "init", {
+      clickmap: true,
+      trackLinks: true,
+      accurateTrackBounce: true
+    });
+  </script>
+  <noscript>
+    <div>
+      <img src="https://mc.yandex.ru/watch/53046907" style="position:absolute; left:-9999px;" alt="" /></div>
+  </noscript>
+  <!-- /Yandex.Metrika counter -->
+</body>
 
-      ym(53046907, "init", {
-        clickmap: true,
-        trackLinks: true,
-        accurateTrackBounce: true
-      });
-    </script>
-    <noscript
-      ><div>
-        <img
-          src="https://mc.yandex.ru/watch/53046907"
-          style="position:absolute; left:-9999px;"
-          alt=""
-        /></div
-    ></noscript>
-    <!-- /Yandex.Metrika counter -->
-  </body>
-</html>
+</html>
```

Comment:
I guess it didn't set up your environment to use EOF (end-of-files). @AmirMehrabi I strongly recommend to you to read carifuly  https://eslint.org/docs/rules/eol-last

Reply:
> I guess it didn't set up your environment to use EOF (end-of-files). @AmirMehrabi I strongly recommend to you to read carifuly https://eslint.org/docs/rules/eol-last

Thanks for the tip. Will do my man.